src/android/CameraLauncher.java

```diff
@@ -526,7 +529,7 @@ private void processResultFromCamera(int destType, Intent intent) throws IOExcep
                 thisJson = gson.toJson(exif);
 
             } catch (IOException e) {
-                e.printStackTrace();
+                TLog.e(TAG, e);
```

Comment:
There are various mysterious blanket catches and suppressions like this in the plugin...